src/a09-dates/verifiers/handler.js

```diff
@@ -0,0 +1,35 @@
+import { getCurrentFormattedDate } from './index.verify.js'
+import { mockCalls } from '../../../verify/test.verify.js'
+import { assertCalls } from '../../../verify/assert.verify.js'
+
+await import('../test/index.test.js')
```

Comment:
can we please use standard imports?

Reply:
Fixed in #15. As soon as it gets merged, I'll update it also here